src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughLastLineCommentCheck.java

```diff
@@ -30,7 +30,7 @@ public void method2(int i) {
             case 1:
                 i++;
                 /* block */ /* fallthru */ // comment
-            case 2:
+            case 2: // violation 'Fall through from previous branch of the switch statement'
                 // this is comment
```

Comment:
here the fall thru comment is not on a last single line

Reply:
There is fallthough comment, but it is in middle of other comments.
Do you mean to pass without  violation it should be :
```
case 1:
                i++;
                /* block */  // comment
                /* fallthru */
            case 2:
```
?

Reply:
Yes
looks like with my changes
```
kevin@inspiron-15-5510:~/Desktop/check_style$ cat Test.java
class Test {

  void methodFallThruCC(int i, int j, boolean cond) {
     while (true) {
         switch (i){
         case 0:
             i++;
            /* block */  // comment
                /* fallthru */
         case 1:
             i++;
             break;
        case 3:
             i++;
            /* block */  /* fallthru */ // comment
         case 4:
             i++;
             break;     

         }
     }
  }
}

kevin@inspiron-15-5510:~/Desktop/check_style$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN" "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="FallThrough">
        <property name="checkLastCaseGroup" value="true" />
        </module>
    </module>
</module>

kevin@inspiron-15-5510:~/Desktop/check_style$ java -jar /home/kevin/Desktop/check_style/checkstyle/checkstyle/target/checkstyle-10.12.3-SNAPSHOT-all.jar -c config.xml Test.java
Starting audit...
[ERROR] /home/kevin/Desktop/check_style/Test.java:16:10: Fall through from previous branch of the switch statement. [FallThrough]
Audit done.
Checkstyle ends with 1 errors.
kevin@inspiron-15-5510:~/Desktop/check_style$ 
```

Reply:
Changes with current update will only accept the fall only on last line

Reply:
Please extend this switch in Input file to show this visually.
I think we can accept this defect, and fix it separate issue.

Reply:
done